src/Xamarin.Android.Tools.AndroidSdk/AndroidAppManifest.cs

```diff
@@ -33,7 +34,7 @@ public class AndroidAppManifest
 			this.doc = doc;
 			manifest = doc.Root;
 			if (manifest.Name != "manifest")
-				throw new ArgumentException ("App manifest does not have 'manifest' root element", nameof (doc));
+				throw new ArgumentException (Resources.ResourceManager.GetString ("XamarinAndroidTools_XAT0001"), nameof (doc));
```

Comment:
> with apologies that this will mean discarding some of this nice work you set up.

Why apologies? It is my job. Sorry that I did not know that exceptions are being kept out of the scope

Started reverting, but will leave in `Resources.resx` for now. Unless you want it differently